pkg/rancher-desktop/components/Help.vue

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+import { shell } from 'electron';
+import Vue from 'vue';
+import { mapState } from 'vuex';
+
+export default Vue.extend({
+  name:       'help',
+  props:      {
+    defaultUrl: {
+      type:     String,
+      default:  null,
```

Comment:
Maybe it's me, but I think `defaultUrl` should actually have a default value that does something reasonable, even if it's `https://docs.rancherdesktop.io/` and we have to manually update it.

Reply:
Hi Eric, the default value is `null` here because I wanted to implement a case when the link button should be disabled:
https://github.com/rancher-sandbox/rancher-desktop/blob/a02b515ba62582d8959c2d3d64a0c39f3620556e/pkg/rancher-desktop/components/Help.vue#L20-L22
https://github.com/rancher-sandbox/rancher-desktop/blob/a02b515ba62582d8959c2d3d64a0c39f3620556e/pkg/rancher-desktop/components/Help.vue#L49-L51

If I put a valid string (url), this mechanism no longer works. Imo, this way is more flexible also.